static/docs/commands-reference/checkout.md

```diff
@@ -165,7 +165,7 @@ This project comes with a predefined HTTP
 [remote storage](/doc/commands-reference/remote). We can now just run `dvc pull`
 that will fetch and checkout the most recent `model.pkl`, `data.xml`, and other
 files that are under DVC control. The model file checksum
-`3863d0e317dee0a55c4e59d2ec0eef33` is specified in the `train.dvc` file:
+`a66489653d1b6a8ba989799367b32c43` is specified in the `train.dvc` file:
```

Comment:
again, this checksum comes from the repo. Indeed if you run it in a different environment you might get a different md5, but let's keep what we have bc it's aligned with the get started repo.